test/admin/draft.test.js

```diff
@@ -0,0 +1,41 @@
+const StitchMongoFixture = require('../fixtures/stitch_mongo_fixture');
+
+import { buildAdminTestHarness, extractTestFixtureDataPoints } from '../testutil';
+
+describe('Draft', () => {
+  let test = new StitchMongoFixture();
+  let th;
+  let appDraft;
+
+  beforeAll(() => test.setup());
+  afterAll(() => test.teardown());
+
+  beforeEach(async() => {
+    const { apiKey, groupId, serverUrl } = extractTestFixtureDataPoints(test);
+    th = await buildAdminTestHarness(true, apiKey, groupId, serverUrl);
+    appDraft = th.app().draft();
+  });
+
+  afterEach(async() => th.cleanup());
+
+  it('responds with a 404 when attempting to get a draft that does not exist', async() => {
```

Comment:
it doesn't look like we have a test for `.get()` in the non-404 case (when the draft does exist) - maybe worth adding that too?

Reply:
oops, duh, totally missed that